classes/class-handler.php

```diff
@@ -432,7 +435,12 @@ private function enqueue_assets( $name, $asset_file, $js_file = null, $css_file
 		if ( $js_file && ! wp_script_is( $name, 'registered' ) ) {
 			$this->register_assets( $name, $asset_file, $js_file, null );
 		}
-		wp_enqueue_script( $name );
+
+		if ( defined( '__EXPERIMENTAL_DYNAMIC_LOAD' ) && 'blocks-everywhere' === $name ) {
```

Comment:
Why do we need the `&& 'blocks-everywhere' === $name`? I think it should be:

```suggestion
		if ( defined( '__EXPERIMENTAL_DYNAMIC_LOAD' ) && $js_file ) {
```

Reply:
Because that is the only file we are actually accounting for. If we or someone else enqueues another asset it could be prevented from being added. With this solution we are only dynamically enqueueing the assets we have accounted for and nothing more.

Reply:
But this will make the `support-content-view` dependency and its assets load synchronously, partially defying the purpose of this change. This function is only used by Blocks Everywhere, it's not part of an API or such.

Reply:
OK I understand now. This makes sense.